### ISSUE TITLE
Right-click to open person in new tab/window gets 404 error

### ISSUE THREAD

**Issue by tjoneslo** [lostcarpark/conclar#141]
This was send to me by one of our users, and I've confirmed it. I've also seen other reload page while on the person pages show the same problem.

- Expand a scheduled item to display its description.
- Right click on a participant's name
- Select Open in a new tab (or open in a new browser window)
-  You get a 404 error

I expect it to do exactly what it does when you left-click -- open a listing for that program participant with a bio and a listing of all of the participant's program items -- just in a new tab or window.

**Comment by mcdemarco** [comment]
This sounds like an issue with the webserver configuration.  If the 404 is the app's internal error page (which doesn't say 404 explicitly), then it may just be an issue in the app configuration, like an omitted package.json edit.

**Comment by tjoneslo** [comment]
https://schedule.readercon.org/people/6566d46e-93df-4407-8c67-f97e49d3d57b
 
 I'm not clear from your documentation what I need to change to make this link show up properly. 

We're using Apache as the server, so specific directions would be helpful.

**Comment by mcdemarco** [comment]
The Apache instructions are in the README.  If you've followed those instructions and are sure mod_rewrite is working, you can try some config tweaks: adding absolute paths for the schedule and people, and making sure the fetch origins are correct.  Both values are near the top of src/config.json.

**Comment by tjoneslo** [comment]
Yes, the Apache mod rewrite instructions fixed the issue. Sorry about the problems. Skipped the part where copy didn't copy the hidden files. 

Thank you .
